pkg/mounter/safe_mounter_windows.go

```diff
@@ -75,9 +76,8 @@ func (mounter *CSIProxyMounter) Mount(source string, target string, fstype strin
 //       rmdir with either pod or plugin context.
 func (mounter *CSIProxyMounter) Rmdir(path string) error {
 	rmdirRequest := &fs.RmdirRequest{
-		Path:    normalizeWindowsPath(path),
-		Context: fs.PathContext_POD,
-		Force:   true,
+		Path:  normalizeWindowsPath(path),
```

Comment:
What's the reason the Context field is no longer set?

Reply:
it doesn't exist in API anymore, there is just one path versus having pod/plugin path being exactly the same https://github.com/kubernetes-csi/csi-proxy/pull/149